.gitignore

```diff
@@ -1,2 +1,8 @@
 target/
-.soroban/
+./.soroban/
+!test.toml
+./cmd/soroban-cli/.soroban/*
+./.soroban/**/*
+.soroban/identities/
+
+cmd/soroban-cli/.soroban/events.json
```

Comment:
Why do we need all these things in the `.gitignore`? Is there something `.soroban/` would miss, or that we want to include in there?

Reply:
My concern is that, if using soroban with `.gitignore` is this difficult, no one will understand how to do it.

Reply:
Not sure why it evolved that way.  Its now just `.soroban` and works as expected.